Ivan_Boichuk_1_HW.cpp

```diff
@@ -0,0 +1,77 @@
+
+
+#include <cmath>
+#include <iostream>
+
+int main() {
+
+  constexpr int errorExitCode{-1};
+  constexpr int successExitCode{};
```

Comment:
я думаю что тут наверное стоит явно указать код возврата, для читабельности. А так во всех случаях когда мы обьявляем переменную то да используйте дефолтную инициализацию
```suggestion
  constexpr int successExitCode{0};
```